src/Administration/Resources/app/administration/src/module/sw-sales-channel/view/sw-sales-channel-detail-base/index.js

```diff
@@ -500,14 +500,27 @@ Component.register('sw-sales-channel-detail-base', {
 
             this.$nextTick(() => {
                 this.deleteSalesChannel(this.salesChannel.id);
-                this.$router.push({ name: 'sw.dashboard.index' });
             });
         },
 
         deleteSalesChannel(salesChannelId) {
             this.salesChannelRepository.delete(salesChannelId, Context.api).then(() => {
                 this.$root.$emit('sales-channel-change');
                 this.salesChannelFavoritesService.refresh();
+
+                this.createNotificationnfo({
```

Comment:
```suggestion
                this.createNotificationInfo({
```

and can you add two jest tests here, where you check that the correct notification is shown?

You can do that by mocking the createNotification calls and asserting the correct one is called with the correct snippet